rules/go_rules.build_defs

```diff
@@ -1034,7 +1034,7 @@ def go_mod_download(name:str, module:str, version:str, test_only:bool=False, vis
     out = name if not go_path_compatibility else f'src/{module}'
     # TODO(jpoole): write a proper please_go tool for this
     cmds = [
-        _set_go_env(),
+        'echo module example.com/module > go.mod', # stops go reading the main go.mod, and downloading all of those too
```

Comment:
```suggestion
        'echo module please_ignore > go.mod', # stops go reading the main go.mod, and downloading all of those too
```

It doesn't have to be a full URL. This is more descriptive IMHO that it's not a test file, it's there on purpose.